src/Mod/Part/App/TopoShapeExpansion.cpp

```diff
@@ -34,34 +34,70 @@
 #include <BRepFill_Generator.hxx>
 #include <BRepTools.hxx>
 #include <BRep_Builder.hxx>
+#include <BRep_Tool.hxx>
+#include <BRepAlgoAPI_BooleanOperation.hxx>
+#include <BRepAlgoAPI_Common.hxx>
+#include <BRepAlgoAPI_Cut.hxx>
+#include <BRepAlgoAPI_Fuse.hxx>
+#include <BRepAlgoAPI_Section.hxx>
+#include <BRepBuilderAPI_Copy.hxx>
+#include <BRepBuilderAPI_MakeEdge.hxx>
+#include <BRepBuilderAPI_MakeWire.hxx>
+#include <BRepCheck_Analyzer.hxx>
+#include <BRepOffsetAPI_MakePipe.hxx>
+#include <ShapeUpgrade_ShellSewing.hxx>
+#include <TopTools_HSequenceOfShape.hxx>
 #include <Precision.hxx>
 #include <ShapeBuild_ReShape.hxx>
+#include <ShapeAnalysis_FreeBounds.hxx>
+#include <BRepTools.hxx>
+
 #include <ShapeFix_Shape.hxx>
 #include <ShapeFix_ShapeTolerance.hxx>
-#include <ShapeUpgrade_ShellSewing.hxx>
 #include <gp_Pln.hxx>
 
 #include <utility>
 #include <BRepBuilderAPI_Copy.hxx>
 #include <BRepBuilderAPI_MakeEdge.hxx>
 #include <TopTools_HSequenceOfShape.hxx>
 #include <ShapeAnalysis_FreeBounds.hxx>
+
+#endif
+
+#if OCC_VERSION_HEX >= 0x070500
+#   include <OSD_Parallel.hxx>
 #endif
 
 #include "TopoShape.h"
+#include "TopoShapeOpCode.h"
 #include "TopoShapeCache.h"
 #include "TopoShapeMapper.h"
 #include "FaceMaker.h"
 
 #include "TopoShapeOpCode.h"
 #include <App/ElementNamingUtils.h>
 #include <BRepLib.hxx>
+#include <OSD_Parallel.hxx>
```

Comment:
This is wrapped in `#if OCC_VERSION_HEX >= 0x070500` above. Also, not in the PCH file?

Reply:
Yeah, superfluous.  Not the only one.  Wow, the IDE difference here is notable.